packages/react/src/generators/init/init.ts

```diff
@@ -63,7 +63,6 @@ function updateDependencies(host: Tree, schema: InitSchema) {
     '@types/react': typesReactVersion,
     '@types/react-dom': typesReactDomVersion,
     '@testing-library/react': testingLibraryReactVersion,
-    'react-test-renderer': reactTestRendererVersion,
```

Comment:
is reactTestRendererVersion still being used?

Reply:
I don't think so.